src/js/controller/controller.js

```diff
@@ -366,16 +368,12 @@ Object.assign(Controller.prototype, {
                 default:
                     return;
             }
-            loadPromise.catch(e => {
-                if (e.code >= 151 && e.code <= 162) {
```

Comment:
@karimJWP The provider errors never reach this point because they are caught in `_setItem`.

Reply:
@karimJWP if we go your route we should:
1. rename`ERROR_LOADING_PROVIDER` to `ERROR_SETTING_ITEM`
2. rename `ERROR_LOADING_PLAYLIST` to `ERROR_CALLING_LOAD`
3. pass either `ERROR_SETTING_ITEM` or `ERROR_CALLING_LOAD` to `_setItem` to establish the correct base code.

Calling load can load a playlist, change the playlist, replace the playlist with a new playlist item, or just set the playlist index, so "ERROR_LOADING_PLAYLIST" isn't really valid base code.